### DISCUSSION TITLE
Plans for the future

### DISCUSSION THREAD

**Discussion by lukaszflorczak** [lukaszflorczak/vue-agile#222] [Announcements]
Hey guys!

After a long break due to more family responsibilities (👶👶) finally, there was an opportunity to resume work on my component. 
I've started with releasing [`v2.0.0`](https://github.com/lukaszflorczak/vue-agile/releases/tag/v2.0.0) version with support for Vue 3.

I have some new features planned. But first of all, I would like to improve the code, using the experiences from the last year of my professional work.

You can find the planned things here: [🛠 TO DO](https://github.com/lukaszflorczak/vue-agile/projects/13), [👾 BUGS](https://github.com/lukaszflorczak/vue-agile/projects/14).

Finally, I would like to thank all the supporters and collaborators and invite you to further cooperation!

**Comment by jesutobi** [comment]
Nice one boss
